contrib/ansible/openshift/README.md

```diff
@@ -33,11 +33,25 @@ Your inventory should have groups of hosts describing the roles they play in the
 [glusterfs]
```

Comment:
openshift-ansible will generate an inventory with glusterfs group, so we don't have an option to rename it to cns.